AWT libraries are not included

GraalVM added support to AWT starting version 21, can you add awt static libs to the linking process

ON Linux I had to relink adding these -lfontmanager -ljavajpeg -llcms -lawt_headless -lawt, and remove -ljavafx_iio to make the linking process succeed.

@alaa-az I added to HelloFX:
```
Desktop.getDesktop();
```
and linked with `awt_xawt`, using `-Djava.awt.headless=false`.

but that failed with:
```
 /usr/bin/ld: $GRAALVM_HOME/lib/static/linux-amd64/glibc/libawt_xawt.a(XToolkit.o): undefined reference to symbol 'XFreeDeviceList'
/usr/bin/ld: /lib/x86_64-linux-gnu/libXi.so.6: error adding symbols: DSO missing from command line
```
So using `awt_xawt` will require adding to the link command other libraries.

For now we are going to include only headless support. A follow-up issue will deal with full AWT support.

--- a/src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java
+++ b/src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java
@@ -58,7 +58,8 @@ public class LinuxTargetConfiguration extends PosixTargetConfiguration {
     private static final List<String> linuxLibs = Arrays.asList("z", "dl", "stdc++", "pthread");
 
     private static final List<String> staticJavaLibs = Arrays.asList(
-            "java", "nio", "zip", "net", "prefs", "j2pkcs11", "sunec", "extnet", "fdlibm"
+            "java", "nio", "zip", "net", "prefs", "j2pkcs11", "sunec", "extnet", "fdlibm",
+            "fontmanager", "javajpeg", "lcms", "awt_headless", "awt"
     );
 
     private static final List<String> staticJvmLibs = Arrays.asList(
@@ -194,14 +195,14 @@ public class LinuxTargetConfiguration extends PosixTargetConfiguration {
                 answer.addAll(linuxfxMedialibs);
             }
         }
-        if (projectConfiguration.getClasspath().contains("javafx-web")) {
+        if (projectConfiguration.hasWeb()) {
             answer.remove(answer.size() - 1);
             answer.addAll(linuxfxWeblibs);
         }
-        answer.addAll(LinuxLinkerFlags.getLinkerFlags());
         if (!crossCompile) {
             answer.addAll(LinuxLinkerFlags.getMediaLinkerFlags());
         }
+        answer.addAll(LinuxLinkerFlags.getLinkerFlags());
         if (usePrismSW || crossCompile) {
             answer.addAll(linuxfxSWlibs);
         }

